docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -23,136 +25,86 @@ import DomainWarning from "@theme/DomainWarning"
 import AjaxWarning from "@theme/AjaxWarning"
 import Tabs from "@theme/Tabs"
 import TabItem from "@theme/TabItem"
+import CodeLinkComparison from "./partials/code-link-comparison.mdx"
```

Comment:
That's not how we do it, please update for both docs.
Also, the imports are not referenced in the documents.

https://github.com/ory/docs#markdown-partials

Reply:
Done, my bad.